cmake/dts.cmake

```diff
@@ -26,6 +26,10 @@ set(DTS_POST_CPP                ${PROJECT_BINARY_DIR}/${BOARD}.dts.pre.tmp)
 # A list of generally accepted vendor prefixes.
 set_ifndef(VENDOR_PREFIXES      ${ZEPHYR_BASE}/dts/bindings/vendor-prefixes.txt)
 
+# Devicetree in CMake.
+set(DTS_CMAKE_SCRIPT            ${ZEPHYR_BASE}/scripts/dts/dts_cmake.py)
```

Comment:
On second thought, can we name this gen_dts_cmake.py, to make it more obvious what it does?

Reply:
fixed